src/test/compile-fail/lifetime-inference-give-expl-lifetime-param.rs

```diff
@@ -49,7 +49,8 @@ struct Baz<'x> {
 
 impl<'a> Baz<'a> {
     fn baz2<'b>(&self, x: &isize) -> (&'b isize, &'b isize) {
-        //~^ HELP consider using an explicit lifetime parameter as shown: fn baz2<'b>(&self, x: &'a isize) -> (&'a isize, &'a isize)
+        //~^ HELP consider using an explicit lifetime parameter as shown: fn baz2<'b>(&self, x: &'
+        // FIXME #34856: The above suggestion is different on Linux and Mac.
```

Comment:
can you open an issue about the fact that this test is unrealiable and put the FIXME with that issue number?